npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker.rb

```diff
@@ -96,13 +96,20 @@ def requirements_update_strategy
       end
 
       def conflicting_dependencies
-        ConflictingDependencyResolver.new(
+        conflicts = ConflictingDependencyResolver.new(
           dependency_files: dependency_files,
           credentials: credentials
         ).conflicting_dependencies(
           dependency: dependency,
           target_version: lowest_security_fix_version
         )
+
+        vulnerable = vulnerability_audit.select do |v|
+          v["fix_available"] == false &&
```

Comment:
I'd make the `fix_available` check a truthy test, too, just to be safe in case the key is missing somehow

```ruby
!v["fix_available"] && v["explanation"]
```